source/manual/schemas.html.md

```diff
@@ -1,41 +1,94 @@
-[student-finance]: https://www.gov.uk/student-finance
+The structured data testing tool is linked to from the [GOV.UK browser extension](https://github.com/alphagov/govuk-browser-extension/pull/133) to help exploration of our implementations.
 
-## [BreadcrumbList][breadcumb-list]
-Used when there is chain of linked Web pages, for example, [attorney generals office][attorney-generals-office]
+The rich result testing tool is occasionally stricter in its requirements than the others because it checks that markup has everything required for Google to present a specially formatted result.
 
-[breadcumb-list]: https://schema.org/BreadcrumbList
-[attorney-generals-office]: https://www.gov.uk/government/organisations/attorney-generals-office
+## Schemas
 
-## [GovernmentOrganization][government-org]
-Used for a governmental organization or agency, for example, [attorney generals office][attorney-generals-office]
+We implement most of our schemas as part of [the machine readable metadata component](https://components.publishing.service.gov.uk/component-guide/machine_readable_metadata) [in GOV.UK publishing components](https://github.com/alphagov/govuk_publishing_components/blob/master/lib/govuk_publishing_components/presenters/schema_org.rb).  This allows us to create a template and share it across all the frontend applications.
 
-[government-org]: https://schema.org/GovernmentOrganization
+Some other schemas are implemented directly in frontend apps. This is usually when:
+
+- the schema is specialised to a particular GOV.UK content type
+- or we're starting to explore a new schema, and iterating using the gem is unnecessary overhead
+
+It is possible to nest schemas within one another. In practice, we don't often do this (preferring links to other content), but we may want to explore de-normalising things in future.
+
+### Article
+
+[The Article schema](https://schema.org/Article) was the first to be implemented on GOV.UK.  We tend to use it as a fallback when there isn't a more specialised schema.
+
+Google indicates that [Article pages may be displayed as a rich result](https://developers.google.com/search/docs/data-types/article) or in a carousel. In practice, I don't think we've seen much difference over standard results for this.  This is possibly because we only supply default images (such as the GOV.UK logo) with the default schema.
+
+### Breadcrumb
+
+This is implemented within [the Breadcrumb component](https://github.com/alphagov/govuk_publishing_components/blob/c0a1add0a222ab105a08133967f0d362e86b5604/app/views/govuk_publishing_components/components/_breadcrumbs.html.erb#L11). This is often shown in Google results, indicating the hierarchy of the site structure leading to the page.
+
+### FAQPage
+
+[The FAQPage schema](https://schema.org/FAQPage) is implemented on [guides](https://www.gov.uk/universal-credit), [answers](https://www.gov.uk/benefits-calculators) and [a transaction](https://www.gov.uk/register-to-vote).
```

Comment:
*a* transaction, or all transactions?

Reply:
It's only [one at the moment](https://github.com/alphagov/frontend/blob/master/app/presenters/machine_readable/transaction_faq_page_schema.rb#L5), because we had a specific need to get the FAQ schema on register to vote. Datalabs have very recently added the GovernmentService schema to all the  transactions, so I sort of wanted to keep that change to a minimum for the moment.